src/rules/newline-after-import.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @fileoverview Rule to enforce new line after import not followed by another import.
+ * @author Radek Benkel
+ */
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function getLineDifference(node, nextToken) {
+  return nextToken.loc.start.line - node.loc.start.line
+}
+
+function ensureNoForbiddenKeyword(context, node, tokenToInspect, tokenValue) {
+  if (getLineDifference(node, tokenToInspect) === 1
+    && tokenToInspect.type === 'Keyword' && tokenToInspect.value !== tokenValue)
+  {
+    context.report({
+      loc: tokenToInspect.loc.start,
+      message: 'Expected empty line after import/require ' +
```

Comment:
Since you know whether it's a `require` or an `import`, might as well indicate which one it is in the error message.